velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -280,7 +280,10 @@ std::unique_ptr<ParquetTypeWithId> ReaderBase::getParquetColumnInfo(
     auto curSchemaIdx = schemaIdx;
     for (int32_t i = 0; i < schemaElement.num_children; i++) {
       ++schemaIdx;
-      auto& childName = schema[schemaIdx].name;
+      auto childName = schema[schemaIdx].name;
+      if (isFileColumnNamesReadAsLowerCase()) {
```

Comment:
At some point we should remove this option from Velox and just let Gluten (and Presto?  Does Presto ever have that option?) to pass in all lower case names in table schema.

Reply:
Sounds good, I don't see this option in presto and this config seems to be added for gluten.

Reply:
In IBM we also use this option with Prestissimo. Parquet data can get created with upper case column names and this option is useful to handle these.